catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/DorisTableOperationsIT.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.doris.integration.test;
+
+import com.datastrato.gravitino.catalog.jdbc.JdbcColumn;
+import com.datastrato.gravitino.catalog.jdbc.JdbcTable;
+import com.datastrato.gravitino.integration.test.util.GravitinoITUtils;
+import com.datastrato.gravitino.rel.TableChange;
+import com.datastrato.gravitino.rel.expressions.NamedReference;
+import com.datastrato.gravitino.rel.expressions.distributions.Distribution;
+import com.datastrato.gravitino.rel.expressions.distributions.Distributions;
+import com.datastrato.gravitino.rel.indexes.Index;
+import com.datastrato.gravitino.rel.types.Type;
+import com.datastrato.gravitino.rel.types.Types;
+import com.google.common.collect.Maps;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("gravitino-docker-it")
+public class DorisTableOperationsIT extends TestDorisAbstractIT {
+  private static final Type VARCHAR_255 = Types.VarCharType.of(255);
+  private static final Type VARCHAR_1024 = Types.VarCharType.of(1024);
+
+  private static Type INT = Types.IntegerType.get();
+
+  private static final String databaseName = GravitinoITUtils.genRandomName("doris_test_db");
+
+  @BeforeAll
+  public static void startup() {
+    TestDorisAbstractIT.startup();
+    createDatabase();
+  }
+
+  private static void createDatabase() {
+    DATABASE_OPERATIONS.create(databaseName, "test_comment", new HashMap<>());
+  }
+
+  private static Map<String, String> createProperties() {
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("replication_allocation", "tag.location.default: 1");
+    return properties;
+  }
+
+  private static void waitForDorisOperation() {
```

Comment:
It's better to use `awaitability` here and we have just introduced it to Gravitino.

Reply:
done